__tests__/app.test.js

```diff
@@ -221,6 +296,90 @@ describe('/api/articles/:id/comments', () => {
                 expect(body.message).toBe('400: Failing schema validation');
             })
         })
+
+        test('404 returns \'404: Not found\' for route that does not exist', () => {
+            return request(app)
+            .get('/api/articles/3/j')
+            .expect(404)
+            .then(({body}) => {
+    
+                expect(body.message).toBe('404: Not found');
+            })
+        })
     })
 })
 
+describe('/api/articles/:id', () => {
+    describe('200: PATCH /api/articles/:id - ', () => {
+        test('200: GET /api/articles - responds with an array of article objects containing the relevant keys', () => {
+            const addToVotes = {
+                inc_votes: 4
+            };
+
+            return request(app)
+            .patch('/api/articles/1')
+            .send(addToVotes)
+            .expect(200)
+            .then(({ body }) => {
+                const { article } = body;
+
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: 1,
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: 104,
+                    article_img_url: expect.any(String),
+                })
+            })
+        })
+    })
+
+    describe('ERRORS', () => {
+        test('400: POST /api/articles/:id/comments - attempting to post a comment without the required fields will return an appropriate error', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .send({})
+            .expect(400)
+            .then(({ body }) => {                
+                expect(body.message).toBe('400: Missing required fields');
+            })
+        })
+
+        test('400: POST /api/articles/:id/comments - attempting to post an incorrectly formatted comment will return an appropriate error', () => {
+            const addToVotes = {
+                inc_votes: 'four'
+            };
+            
+            return request(app)
+            .patch('/api/articles/1')
+            .send(addToVotes)
+            .expect(400)
+            .then(({ body }) => {                
+                expect(body.message).toBe('400: Failing schema validation');
+            })
+        })
+
+        test('404 returns \'404: Not found\' for route that does not exist', () => {
+            return request(app)
```

Comment:
A bad path test would likely have auto passed? This isn't related to the functionality of this endpoint and so the test block doesn't belong in this describe block. This bad path test is only needed once in the entire test suite as that's checking the one app.all('*') or app.use('*') you have setup.